indico/modules/events/tracks/templates/management.html

```diff
@@ -1,20 +1,25 @@
 {% extends 'layout/full_content_base.html' %}
+{% from 'events/tracks/_track_list.html' import render_track_list %}
 
 {% block title %}
     {%- trans %}Programme{% endtrans -%}
 {% endblock %}
 
-{% block content %}
-    {% for track in tracks %}
-        <div>
-            {{ track.title }}
-        </div>
-    {% endfor %}
+{% block description %}
+    {%- trans %} Tracks form the programme of the event. Here you can find a list of tracks for this event.
+        You can add, edit, remove or sort these tracks.
+    {% endtrans -%}
+{% endblock %}
 
+{% block content %}
     <a class="i-button highlight icon-plus"
         data-href="{{ url_for('.create', event) }}"
+        data-title="{% trans %}Add a new track{% endtrans %}"
         data-ajax-dialog
         data-reload-after>
         {% trans %}Add track{% endtrans %}
     </a>
```

Comment:
FWIW, the whitespace usually does not matter on the outside. So it's up to you whether you want to strip it or not (we don't care whether the html output is pretty or not).

However, on the inside of a trans it does matter (if there's whitespace - here there's none anyway) since those messages get extracted to the translation dict and having surrounding whitespace there is quite awful